barretenberg/cpp/src/barretenberg/ecc/fields/field_conversion.hpp

```diff
@@ -195,36 +166,26 @@ template <std::integral T> std::vector<bb::fr> inline convert_to_bn254_frs(const
     return fr_vec;
 }
 
-std::vector<bb::fr> inline convert_to_bn254_frs(const grumpkin::fr& val)
-{
-    auto fr_arr = convert_grumpkin_fr_to_bn254_frs(val);
-    std::vector<bb::fr> fr_vec(fr_arr.begin(), fr_arr.end());
-    return fr_vec;
-}
+/**
+ * @brief Converts grumpkin::fr to 2 bb::fr elements
```

Comment:
I'm slightly torn on this because I generally think more documentation is better but I think it's best to only add this level of description to either the declaration OR the definition, not both. It's a bit less clutter that way and it also avoids the two things diverging and causing confusion about the source of truth. Excellent comments here - but maybe only need them in one place

Reply:
will put the comment in the definition then since it has implementation details